README.md

```diff
@@ -30,12 +30,23 @@ Requirements:
 
 - go 1.6 or higher
 - A [working golang](https://golang.org/doc/code.html) environment
+- A [properly configured](https://golang.org/doc/code.html#Organization) Go workspace
 
-From the project root directory, run:
+In your terminal, run:
 ```
+cd $GOPATH
```

Comment:
@raykrueger TLDR; for most people (I did read it though and I found the absolute minimum (which is probably no hard to add) would be the two lines (correct me if I'm wrong) that I pasted. )